spec/datadog/tracing/contrib/elasticsearch/transport_spec.rb

```diff
@@ -10,8 +10,8 @@
 
 RSpec.describe 'Elasticsearch::Transport::Client tracing' do
   before do
-    WebMock.enable!
-    WebMock.disable_net_connect!
+    call_web_mock_function_with_agent_host_exclusions { |options| WebMock.enable! options }
+    call_web_mock_function_with_agent_host_exclusions { |options| WebMock.disable_net_connect! options }
```

Comment:
Is `call_web_mock_function_with_agent_host_exclusions` here because one would run this test case without having the agent running locally? If so, that's not a concern as these tests are not meant to successfully run with an agent running.

Reply:
This is used to allow http web traffic to the test agent, which would otherwise be disabled by webmock for these tests.